EnvironmentSetup.ps1

```diff
@@ -68,7 +68,7 @@ if ($microbuild -or ($vstarget -eq "15.0" -and -not $skipRestore)) {
     Write-Output ""
     Write-Output "Installing Nuget MicroBuild packages"
 
-    & "$rootdir\Nodejs\.nuget\nuget.exe" restore "$rootdir\Nodejs\Setup\swix\packages.config" -PackagesDirectory "$packagedir"
+    & "$rootdir\Nodejs\.nuget\nuget.exe" restore "$rootdir\Nodejs\Setup\swix\project.json" -PackagesDirectory "$packagedir" -ConfigFile "$rootdir\Nodejs\.nuget\nuget.config"
```

Comment:
This works with the nightly build, however for a local build you should set `-skipRestore` to `true`. This won't prevent you from building the product, but you won't be able to build the setup packages we need for the VS insertion.